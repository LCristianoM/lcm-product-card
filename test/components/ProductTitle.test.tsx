import React from 'react';
import { ProductTitle } from '../../src/components/ProductTitle';
import TestRenderer  from 'react-test-renderer';
import { product1 } from '../data/products';
import { ProductCard } from '../../src/components/ProductCard';



describe('ProductTitle', () => {
  test('debe mostrar el componete correctamene con el titulo personalizado', () => {
    const wrapper = TestRenderer.create(
      <ProductTitle title="Custom Produc" />
    )
    expect(wrapper.toJSON()).toMatchSnapshot();
  })
});

test('debe mostrar el componete con el nombre del producto', () => {

  const wrapper = TestRenderer.create(
      <ProductCard product={ product1} >
        {
          () => (
            <ProductTitle />
          )
        }

      </ProductCard>
    )
    expect(wrapper.toJSON()).toMatchSnapshot();
  
})
