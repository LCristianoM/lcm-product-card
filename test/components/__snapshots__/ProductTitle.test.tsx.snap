// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle debe mostrar el componete correctamene con el titulo personalizado 1`] = `
<span
  className="productDescription undefined"
>
  Custom Produc
</span>
`;

exports[`debe mostrar el componete con el nombre del producto 1`] = `
<div
  className="productCard undefined "
>
  <span
    className="productDescription undefined"
  >
    Coffe Mug - Card
  </span>
</div>
`;
